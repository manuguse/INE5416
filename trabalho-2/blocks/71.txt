1 1 6 6 6 12 12 15
2 2 7 6 10 12 15 15
3 2 2 2 10 10 15 15
3 5 2 10 10 13 13 13
3 5 8 8 10 13 13 13
3 4 4 8 9 9 16 16
4 4 9 9 9 9 16 17
4 4 4 11 9 14 14 17