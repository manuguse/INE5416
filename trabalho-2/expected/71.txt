1 2 3 4 1 2 3 5
2 3 1 2 6 1 2 4
4 1 5 6 5 2 1 3
3 2 4 1 4 6 5 2
2 1 3 2 3 4 3 1
1 6 5 1 4 7 2 3
7 2 1 5 3 6 1 2
3 1 4 1 2 1 2 1